scipy/stats/_morestats.py

```diff
@@ -3957,14 +3960,17 @@ def directionalmean(samples, *, axis=0, normalize=True):
     Examples
     --------
     >>> from scipy.stats import directionalmean
-    >>> data = np.array([[0.8660254, 0.5, 0.], [0.8660254, -0.5, 0.]])
+    >>> data = np.array([[3, 4],    # first observation, 2D vector space
+    ...                  [6, -8]])  # second observation
     >>> directionalmean(data)
-    array([1., 0., 0.])
+    array([1., 0.])
 
-    The *regular* sample mean in contrast does not lie on the unit sphere.
+    In contrast, the regular sample mean of the vectors would be influenced
+    by the magnitude of each observation. Furthermore, the result would not be
```

Comment:
Perhaps more important than the fact that the _output_ of `np.mean` is not normalized is that the magnitudes of the _input_ don't matter.